afkak/brokerclient.py

```diff
@@ -30,7 +31,7 @@
 
 MAX_RECONNECT_DELAY_SECONDS = 15
 INIT_DELAY_SECONDS = 0.1
-CLIENT_ID = "a.kbc"
+CLIENT_ID = b"a.kbc"  # XXX Why is this not the same as KafkaClient's default?
```

Comment:
Because then if KafkaClient doesn't pass it's CLIENT_ID, you get a different value and you can see a difference in behavior based on the different args :-)